rfcs/0000-rfc-template.md

```diff
@@ -22,6 +23,37 @@ Stage X: Provide a brief explanation of why the proposal is being marked as aban
 -->
 
 ## Fields
+The `vulnerabilities` fields being proposed are as follows:
+
+Field | Type | Description /Use Case
+vulnerability.vendor.id | keyword | A vulnerability doesn't have necessary a CVE associated with it. I don't know if it makes sense to seperate vulnerability ID (like CVEs) to the vendor/detection IDs.
+vulnerability.title	| keyword | Title/Name/Short Description for vulnerability, to be used in flyout and dashboards.
+vulnerability.mitigation | text | Explains user how to fix or mitigate the problem, could be usefd to store resolution from the scanner vendor or document mitigation in place
+vulnerability.published	| date | The “published” field indicates the date when information about a specific vulnerability was publicly disclosed or made available.It represents the moment when details about the vulnerability were shared with the security community, vendors, and the public.This field helps security professionals track the timeline of vulnerability awareness.Example value: If a critical vulnerability (e.g., CVE-2024-1234) affecting a widely used software is disclosed on February 1, 2024, the published date for that CVE would be February 1, 2024.
```

Comment:
For vulnerability events, would this be `@timestamp`?

> This is the date/time extracted from the event, typically representing when the event was generated by the source.

Although I'm not familiar with how third party systems publishes these types of events.

Reply:
@ebeahan no, this is different. The published vulnerability time/date is the same for everyone. There are 2 "common" dates:
- reserved: when the vendor is "booking" a CVE number. Some companies are doing it in batches, it doesn't mean much.
- published: as explained in the RFC, it's when details is made available: which vulnerability on which software.